src/preload/overrides.c

```diff
@@ -242,6 +242,12 @@ uid_t geteuid(void) {
 #endif
 }
 
+static void libstdcpp_not_found(void) {
+  logmsg("[rr] Interposition for libstdc++ called but symbol lookups into libstdc++ failed.\n"
```

Comment:
Does this have to be using syscallbuf code? I know this is in the error path already but this also has he potential of causing an assertion. Ref https://github.com/rr-debugger/rr/pull/3284

Reply:
Yeah I think we should use a dedicated write syscall here.